src/pages/Content/index.js

```diff
@@ -13,3 +13,11 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
+function playRecording(recordedEvents) {
+  recordedEvents.forEach(element => {
+    if(element.type == "click") {
+      document.querySelector(element.selector).click();
```

Comment:
We have click function and I think we should invoke that function here.

Reply:
I couldn't find the element.type attribute in the Element document https://developer.mozilla.org/en-US/docs/Web/API/Element. Is it correct to check element.type this way? Can you please share the document with me? I would like to learn what this is doing?

Reply:
I hope @YukiChung read my comment. So resolving this conversation to make it easier for Agam to review.